.github/workflows/deploy.yml

```
@@ -8,21 +8,29 @@ on:
 permissions:
   contents: write
 
+env:
+  image: ryanmillard/img2num-dev:latest
+
 jobs:
   cmake-build:
     name: Build WASM
     uses: ./.github/workflows/cmake-build.yml
+    with:
+      image: ${{ env.image }}
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)